superset-frontend/src/dashboard/components/PropertiesModal/index.tsx

```diff
@@ -161,10 +161,15 @@ const PropertiesModal = ({
 
       form.setFieldsValue(dashboardInfo);
       setDashboardInfo(dashboardInfo);
-      setJsonMetadata(metadata ? jsonStringify(metadata) : '');
       setOwners(owners);
       setRoles(roles);
       setColorScheme(metadata.color_scheme);
+
+      // temporary fix to remove positions from dashboards' metadata
```

Comment:
This is to ensure that dashboards that are being affected by an excessively large positions object should work properly. This check can be likely removed in the near future.